knowledge/doc.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2023 IndyKite
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+Package knowledge implements the IndyKite Identity Knowledge API Client.
+*/
+package knowledge
+
+const versionClient = "v0.16.0" // x-release-please-version
```

Comment:
Should this version be here?

Reply:
I really don't know if I need to add that manually the first time, but I know that the release please workflow automatically updates it on new releases